science.py

```diff
@@ -95,36 +97,33 @@ def init_motors(bus: can.Bus):
 
 
 async def key_pressed(args, bus: can.Bus, key: str):
-    global first_cup_idx
     if args.debug:
         print(f"Pressed: {key}")
-    if key == "up" or key == "down":
-        power = DRILL_ARM_POWER * (1 if key == "up" else -1)
+    if key == "w" or key == "s":
+        power = DRILL_ARM_POWER * (1 if key == "w" else -1)
         set_motor_power(bus, DRILL_ARM_SERIAL, power)
-    elif key == "w" or key == "s":
-        power = DRILL_POWER * (1 if key == "w" else -1)
+    elif key == "space" or key == "z":
+        power = DRILL_POWER * (1 if key == "space" else -1)
         set_motor_power(bus, DRILL_SERIAL, power)
+    elif key == "left" or key == "right":
+        pos = 90 + PANO_SPEED * (1 if key == "right" else -1)
```

Comment:
This is hack, since the pano servo was a continuous servo plugged into a positional servo port, so setting 90deg => speed 0. Setting a position >90 is positive speed, <90 is negative speed. The correct way to do this is to plug the servo into a continuous servo port on the science board and use the continuous servo speed set packet.